.scala-steward.conf

```
@@ -0,0 +1,4 @@
+# If set, Scala Steward will use this message template for the commit messages and PR titles.
+# Supported variables: ${artifactName}, ${currentVersion}, ${nextVersion} and ${default}
+# Default: "${default}" which is equivalent to "Update ${artifactName} to ${nextVersion}" 
+commits.message = "chore(deps): Update ${artifactName} from ${currentVersion} to ${nextVersion}"
```

```suggestion
commits.message = "build(deps): update ${artifactName} from ${currentVersion} to ${nextVersion}"
```

To be more in line with conventional commits